packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/scroll/ReactScrollViewManager.java

```diff
@@ -379,4 +379,22 @@ public void setPointerEvents(ReactScrollView view, @Nullable String pointerEvent
   public void setScrollEventThrottle(ReactScrollView view, int scrollEventThrottle) {
     view.setScrollEventThrottle(scrollEventThrottle);
   }
+
+  @ReactProp(name = "inverted")
```

Comment:
We'll need to add this prop to ScrollViewProps for this to work on Fabric (which would be useful, because we need inverted for Windows too :)

https://github.com/facebook/react-native/blob/main/packages/react-native/ReactCommon/react/renderer/components/scrollview/ScrollViewProps.h